manual/_static/eqno.js

```
@@ -0,0 +1,52 @@
+\* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ */
+
+document.addEventListener("DOMContentLoaded", () => {
+  /** @param {Node} node */
+  const beautifyEquationNumbers = (node) => {
+    // Regular expression to match: `(123)`.
+    const regex = /\((\d+)\)/g;
+
+    // Prettify the equation numbers.
+    let match;
+    let text = node.nodeValue;
+    while ((match = regex.exec(text)) !== null) {
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)